_data/travel-steps.yml

```diff
@@ -15,5 +15,5 @@ first_time_steps:
     href: travel-and-leave/travel-and-leave-policies/first-time-travel-travel-card/
   - text: Update Concur profile
     href: travel-and-leave/travel-and-leave-policies/first-time-travel-complete-concur-profile/
-  - text: Start approval process
+  - text: "Book Travel"
```

Comment:
probably should be lowercase t - Book travel

Reply:
updated!